graphhopper.sh

```diff
@@ -205,104 +92,15 @@ BASENAME=$(basename "${FILE}")
 # NAME = file without extension if any
 NAME="${BASENAME%.*}"
 
-if [ "$FILE" == "-" ]; then
-   OSM_FILE=
-elif [ ${FILE: -4} == ".osm" ] || [ ${FILE: -4} == ".xml" ] || [ ${FILE: -4} == ".pbf" ]; then
-   OSM_FILE="$FILE"
-elif [ ${FILE: -7} == ".osm.gz" ]; then
-   OSM_FILE="$FILE"
-elif [ ${FILE: -3} == "-gh" ]; then
-   OSM_FILE="$FILE"
-   NAME=${FILE%%???}
-elif [ ${FILE: -4} == ".ghz" ]; then
-   OSM_FILE="$FILE"
-   if [[ ! -d "$NAME-gh" ]]; then
-      cd $DATADIR && unzip "$BASENAME" -d "$NAME-gh"
-   fi
-else
-   # no known end -> no import
-   OSM_FILE=
-fi
-
-LINK=$(echo $NAME | tr '_' '/')
-if [ "$FILE" == "-" ]; then
-   LINK=
-elif [ ${FILE: -4} == ".osm" ]; then
-   LINK="http://download.geofabrik.de/$LINK-latest.osm.bz2"
-elif [ ${FILE: -4} == ".ghz" ]; then
-   LINK="https://graphhopper.com/public/maps/0.1/$FILE"
-elif [ ${FILE: -4} == ".pbf" ]; then
-   LINK="http://download.geofabrik.de/$LINK-latest.osm.pbf"
-else
-   # e.g. if directory ends on '-gh'
-   LINK="http://download.geofabrik.de/$LINK-latest.osm.pbf"
-fi
-
 : "${JAVA_OPTS:=-Xmx1000m -Xms1000m}"
```

Comment:
```suggestion
: "${JAVA_OPTS:=-Xmx1g -Xms1g -Ddw.server.application_connectors[0].bind_host=0.0.0.0 -Ddw.server.application_connectors[0].port=8989}"
```

Make it literally the same as in the Dockerfile

Reply:
I changed the dockerfile, I think it's the right direction. This needs to be tested of course.